openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -709,6 +734,17 @@ def _validate_email_doesnt_exist(email):
         raise errors.AccountEmailAlreadyExists(_(accounts.EMAIL_CONFLICT_MSG).format(email_address=email))
 
 
+def _validate_secondary_email_doesnt_exist(email):
```

Comment:
`_validate_secondary_email_doesnt_exist` -> `_validate_secondary_email_not_exists` or maybe `_validate_secondary_email_value` (considering the generic nature of email value lookup)

Reply:
I think this name better adheres to the naming convention of the file. e.g `_validate_username_doesnt_exist`, `_validate_email_doesnt_exist`. Also, this is not a generic function as it only validates that no user profile exists with the secondary email of given value.